spec/models/service_template_transformation_plan_request_spec.rb

```diff
@@ -5,6 +5,32 @@
       ServiceResource.new(:resource => vm, :status => status)
     end
   end
+
+  let(:dst_ems) { FactoryGirl.create(:ext_management_system) }
```

Comment:
Any reason not to move these `let`s to be inside Line57?

Reply:
Done.